docs/src/app/components/pages/components/Dialog/ExampleCustomWidth.jsx

```diff
@@ -28,12 +28,12 @@ export default class DialogExampleCustomWidth extends React.Component {
     const actions = [
       <FlatButton
         label="Cancel"
-        secondary={true}
+        primary={true}
         onTouchTap={this.handleClose}
       />,
       <FlatButton
         label="Submit"
-        primary={true}
+        secondary={true}
```

Comment:
I'm having second thought, what about using `primary` here too to keep the color consistent?